source/release-notes/4-0-2.rst

```diff
@@ -0,0 +1,102 @@
+.. index:: CrafterCMS 4.0.2 Release Notes
+
+----------------
+CrafterCMS 4.0.2
+----------------
+
+*Released 2022*
+
+This release includes fixes for issues with the release version 4.0.1
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* New publishing on demand option to ``Publish everything`` |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5682>`__ for more information
+
+  For more information on the new publishing on demand option, see :ref:`publish-everything`
+
+* Items in an item selector now displays the item's path in the form control |br|
+  See `here <https://github.com/craftercms/craftercms/issues/5696>`__ for more information
+
+
+^^^^^^^
+Updates
```

Comment:
We also:
* Renamed the `dropTargets` data source to `components`
* Enhanced Crafter Studio clustering


 (you should also update this https://docs.craftercms.org/en/4.0/developers/content-modeling.html?#list-of-data-sources)